app/javascript/react/utils/createGraphData.ts

```diff
@@ -0,0 +1,19 @@
+import { LatLngLiteral } from "../types/googleMaps";
+
+export const createFixedSeriesData = (data: any[]) =>
```

Comment:
you can improve the typing here, looking at the code it should be: `{ time: number; value: number };`